beluga_tutorial/README.md

```diff
@@ -0,0 +1,505 @@
+# Simple MCL Implementation Using Beluga
+
+- [Simple MCL Implementation Using Beluga](#simple-mcl-implementation-using-beluga)
+  - [Overview](#overview)
+  - [Requirements](#requirements)
+  - [Monte Carlo Localization (MCL) Background](#monte-carlo-localization-mcl-background)
+  - [Tasks](#tasks)
+  - [1 Create the Workspace](#1-create-the-workspace)
+  - [2 Create the Package](#2-create-the-package)
+  - [3 Write the Simulation code](#3-write-the-simulation-code)
+    - [3.1 Tutorial Parameters](#31-tutorial-parameters)
+    - [3.2 Landmark Map](#32-landmark-map)
+    - [3.3 Generate Particles](#33-generate-particles)
+    - [3.4 Simulation Loop](#34-simulation-loop)
+    - [3.5 Control Update - Motion Model](#35-control-update---motion-model)
+    - [3.6 Measurement Update - Sensor Model](#36-measurement-update---sensor-model)
+    - [3.7 Resample](#37-resample)
+  - [4 Compile and run code](#4-compile-and-run-code)
+    - [4.1 package.xml](#41-packagexml)
+    - [4.2 CMakeLists.txt](#42-cmakeliststxt)
+    - [4.2 Build and run](#42-build-and-run)
+
+## Overview
+![Simulation Video](./resources/simulation_video.mp4)
```

Comment:
@Alondruck meta: to have this in main documentation, we either make it a GIF, optimized in size, or we use https://github.com/sphinx-contrib/video. The GIF alternative may give us faster load times.

Edit: it's a 24.5MB video, it's huge :open_mouth:

Reply:
Yes, sorry I knew that is not correct but I left this here because I don't wanted to forget :sweat_smile:. The GIF is a good idea!